kvbc/src/Replica.cpp

```diff
@@ -323,8 +323,25 @@ void Replica::createReplicaAndSyncState() {
   handleNewEpochEvent();
   handleWedgeEvent();
   bftEngine::impl::DbCheckpointManager::instance().initializeDbCheckpointManager(
-      m_dbSet.dataDBClient, m_replicaPtr->persistentStorage(), aggregator_, [this]() -> uint64_t {
-        return getLastBlockId();
+      m_dbSet.dataDBClient,
+      m_replicaPtr->persistentStorage(),
+      aggregator_,
+      [this]() -> uint64_t { return getLastBlockId(); },
+      [](BlockId blockIdAtCheckpoint, const std::string &path) {
```

Comment:
blockIdAtCheckpoint name should be block_id_at_checkpoint

Reply:
Thanks, fill fix.